app/views/casework/decisions/interests_of_justices/edit.html.erb

```diff
@@ -0,0 +1,31 @@
+<% title t '.title' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body-l govuk-!-margin-bottom-2">
+      <%= @crime_application.reference %>
+    </p>
+
+    <h2 class="govuk-heading-xl govuk-!-margin-bottom-4">
+      <%= @crime_application.applicant.full_name %>
+    </h2>
+
+    <h1 class="govuk-heading-xl">
+      <%= t '.title' %>
+    </h1>
+
+    <%= form_with model: @form_object,
+                  url: crime_application_decision_interests_of_justice_path, method: :put do |f| %>
+
+      <%= f.govuk_error_summary %>
+
+      <%= f.govuk_collection_radio_buttons :result,
+                                           @form_object.possible_results,
+                                           :to_s %>
+      <%= f.govuk_text_area :details %>
+      <%= f.govuk_text_field :assessed_by, autocomplete: 'off' %>
+      <%= f.govuk_date_field :assessed_on, maxlength_enabled: true, autocomplete: 'off' %>
```

Comment:
~~Do we need to apply any validation on this date field?~~
Edit: spoke to tim, whether this field populates in a date from maat is being discussed with ucd so no change for now